### ISSUE TITLE
p1_NLCD_data_zipped gets outdated after first run through pipeline

### ISSUE THREAD

**Issue by jds485** [USGS-R/drb-inland-salinity-ml#61]
After a 1st `tar_make()`, three NLCD targets are outdated (image below). When a 2nd `tar_make()` is called, the first two (zipped and unzipped) are rebuilt, and the last is skipped. After that, `tar_make()` skips all targets. So, I'm expecting that something in the `p1_NLCD_data` function is causing the previous two NLCD targets to become outdated. I did not see anything that would cause it, though.
![image](https://user-images.githubusercontent.com/8761242/150605738-05b35301-7edf-4f42-9102-3ea2984bd258.png)

**Comment by jds485** [comment]
Noting that this should be addressed by the suggested ScienceBase date last modified check in #98.

**Comment by msleckman** [comment]
is this still happening btw?

**Comment by jds485** [comment]
yes. it happened yesterday when I tested building the full pipeline in the container

**Comment by jds485** [comment]
I figured out why this is happening: The `p1_NLCD2011_data_unzipped` target writes an `unzipped` directory within the `p1_NLCD2011_data_zipped` output directory, so that causes `p1_NLCD2011_data_zipped` to become outdated.
